app/models/post.rb

```diff
@@ -0,0 +1,12 @@
+class Post < ActiveRecord::Base
+  belongs_to :user
```

Comment:
- [ ] As per the given _ERD_, the _Post_ **association** should have _Post_ belong to _User_ **through** _Author_, an alias of _User_, like below:
`belongs_to :author, class_name: 'User'` 

    Kindly ensure to improve upon this aspect.